lib/ansible/modules/cloud/amazon/ec2_lc.py

```diff
@@ -188,11 +188,23 @@
 
 def create_block_device_meta(module, volume):
     MAX_IOPS_TO_SIZE_RATIO = 30
+
+    # device_type has been used historically to represent volume_type,
+    # however ec2_vol uses volume_type, as does the BlockDeviceType, so
+    # we add handling for either/or but not both
+    if 'device_type' in volume and 'volume_type' in volume:
+        module.fail_json(msg='device_type is a deprecated name for volume_type. '
```

Comment:
This also needs to warn if the deprecated name, `device_type`, is used on its own. Otherwise, people will silently just use device_type without knowing about its deprecation.

Reply:
Ok, added a deprecation message for ec2_lc and also ec2